test/api/v3/unit/libs/analyticsService.test.js

```diff
@@ -101,6 +101,18 @@ describe('analyticsService', () => {
               amplitudeNock.done();
             });
         });
+
+        it('logs unknown if headers are not passed in', () => {
+          amplitudeNock
+            .filteringpath(/httpapi.*platform.*unkown.*/g, '');
```

Comment:
Thanks, that's what I get for writing this in the 5 minutes before I go to bed without actually running the tests